packages/dates/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## Unreleased -->
 
+## 1.1.0 - 2022-01-20
+
+- Added format function, matching `moment().format`
```

Comment:
```suggestion
## Unreleased

### Added

- Added `format()` function, matching `moment().format`. [[#2135](https://github.com/Shopify/quilt/pull/2135)]
```

Don't add the version number stuff, that gets added as part of the release.
Add the PR reference in the change.

Reply:
Ah, thanks for letting me know how it should be done 👍